src/scripts/farming/Farming.ts

```diff
@@ -205,7 +235,10 @@ class Farming implements Feature {
             20,
             2,
             [0, 10, 10, 0, 0],
+            20,
             BerryColor.Purple,
+            10.8,
```

Comment:
Bluk is 4.7" on your link, but displays as 4.3 with this size

Reply:
Seems as though that link wasn't entirely reliable. Bulbapedia, Serebii, and its in-game Berry Tag in both gen 3 and 4 list it as 4.3.